src/sdk/pynni/nni/evolution_tuner/evolution_tuner.py

```diff
@@ -18,61 +18,35 @@
 # DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 """
-evolution_tuner.py including:
-    class OptimizeMode
-    class Individual
-    class EvolutionTuner
+evolution_tuner.py
 """
 
 import copy
-from enum import Enum, unique
 import random
 
 import numpy as np
-
 from nni.tuner import Tuner
-from nni.utils import extract_scalar_reward
-from .. import parameter_expressions
-
-@unique
-class OptimizeMode(Enum):
-    """Optimize Mode class
+from nni.utils import (NodeType, OptimizeMode, extract_scalar_reward,
+                       split_index)
 
-    if OptimizeMode is 'minimize', it means the tuner need to minimize the reward
-    that received from Trial.
-
-    if OptimizeMode is 'maximize', it means the tuner need to maximize the reward
-    that received from Trial.
-    """
-    Minimize = 'minimize'
-    Maximize = 'maximize'
-
-
-@unique
-class NodeType(Enum):
-    """Node Type class
-    """
-    Root = 'root'
-    Type = '_type'
-    Value = '_value'
-    Index = '_index'
+from .. import parameter_expressions
 
 
-def json2space(x, oldy=None, name=NodeType.Root.value):
+def json2space(x, oldy=None, name=NodeType.ROOT):
```

Comment:
can you write some unit-test for the util function? Such as function "json2space", "json2parameter" and so on....

Reply:
I add config test instead of unit-test just.